README.md

```diff
@@ -1,70 +1,54 @@
 
-## Available Scripts
+## Built With &hearts;
 
-In the project directory, you can run:
+- ReactJs
+- Redux
+- Redux-thunk
+- Axios
+- Bootstrap
 
-### `npm start`
+### Prerequisites
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Good knowledge on javascript objects
+- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Node package. [Link to download node](https://nodejs.org/en/download/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting started
 
-### `npm test`
+to run the project locally, clone the project using the command
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+`git clone https://github.com/Daphineatim/covid-data-app.git`
 
-### `npm run build`
+###  Live link
+`https://covid19-capstone.netlify.app`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Run test
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+navigate to the project directory, from GitHub,
+to test or run the application double click on the `index.html` file to open in your browser
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Authors
 
-### `npm run eject`
+👤 **Daphine Atim**
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+-GitHub: @Daphineatim
+-Twitter: @dhaphyn
+-LinkedIn: DaphineAtim
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 🤝 Contributing
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Contributions, issues, and feature requests are welcome!
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Feel free to check the [issues page](../../issues/).
 
-## Learn More
+## Show your support
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Give a ⭐️ if you like this project!
 
-To learn React, check out the [React documentation](https://reactjs.org/).
-
```

Comment:
- [ ] It would be better if we add a link to our video presentation as seen in the screenshot below

![Screenshot from 2022-11-29 09-56-21](https://user-images.githubusercontent.com/46388113/204498078-bea44655-9c62-48d5-9ab5-17f79bbfd07b.png)